ambry-store/src/main/java/com/github/ambry/store/BlobStore.java

```diff
@@ -1183,7 +1183,7 @@ public boolean isEmpty() {
 
   @Override
   public boolean isBootstrapInProgress() {
-    return (new File(dataDir, BOOTSTRAP_FILE_NAME)).exists();
```

Comment:
nit: We can now remove `String BOOTSTRAP_FILE_NAME` prop from this class altogether.

Reply:
Done